app/furniture/tobias/trusts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Tobias
+  class TrustsController < FurnitureController
+    expose :trust, model: Trust
+
+    def show
+      authorize trust
```

Comment:
The `authorize` method comes from `Pundit`, and is how we link the code in `TrustPolicy`, which describes who is allowed to see or change the `Trust` model, to the action being taken by the user.